packages/core/src/amazonqFeatureDev/controllers/chat/controller.ts

```diff
@@ -144,7 +145,7 @@ export class FeatureDevController {
         })
     }
 
-    private async processChatItemVotedMessage(tabId: string, messageId: string, vote: string) {
+    private async processChatItemVotedMessage(tabId: string, _messageId: string, vote: string) {
```

Comment:
probably just makes sense to remove it rather then underscore it if its not being used, but that can be a different PR